projects/samples/contests/robocup/protos/WhiteGrass.proto

```diff
@@ -0,0 +1,27 @@
+#VRML_SIM R2021a utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# A grass material containing several grass types.
```

Comment:
This description is wrong because this PROTO doesn't contain several grass types.